src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/GenotypingEngine.java

```diff
@@ -353,56 +351,38 @@ private OutputAlleleSubset(final List<Allele> alleles, final List<Integer> mleCo
      * @return information about the alternative allele subsetting {@code null}.
      */
     private OutputAlleleSubset calculateOutputAlleleSubset(final AFCalculationResult afCalculationResult, final VariantContext vc) {
```

Comment:
@droazen Is correct. It was intended to maintain state for upstream deletions. Make sure the upstream spanning deletion does not span shard barriers, then there will be problems with the book keeping. AFAIK, shards overlap genome locations but for a large spanning deletion, the size might not be sufficient.